datatypes/src/operations/image/colorizer.rs

```diff
@@ -9,6 +9,19 @@ use std::collections::HashMap;
 use std::convert::TryFrom;
 use std::str::FromStr;
 
+#[derive(Copy, Clone, Debug, Deserialize, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+#[serde(untagged)]
+pub enum ColorFields {
```

Comment:
Maybe just call it `DefaultColors`, since the no data color and breakpoints are also color fields.

Reply:
Changed with https://github.com/geo-engine/geoengine/pull/684/commits/86ec9cd2db7b3f5fdc9ce59ee03ac2a652c072b7